src/pages/studies/SearchBar.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
+import { VisualType } from '@ferlab/ui/core/components/filters/types';
+import { ISqonGroupFilter } from '@ferlab/ui/core/components/QueryBuilder/types';
+import { Select, Tag } from 'antd';
+
+import history from 'services/history';
+
+import { ItemProps } from './SidebarFilters';
+import { updateFilters } from './utils';
+
+import styles from './SearchBar.module.scss';
+
+type OwnProps = {
+  options: ItemProps[];
+  filters: ISqonGroupFilter;
+};
+
+const extractCodesFromFilter = (filters: ISqonGroupFilter) => {
```

Comment:
code is plural in the func's name. However, looking at the javascript lines it extracts only one code. What is it that is desired exactly?

Reply:
Suggestions:

-  write a type (s: { content: { field: string } }) 

- find?.content?.value || []

Reply:
well. under the field key `code` there is an array of codes. So the plural makes sence. since it is that array of codes that i return. Unless there is no filter `code`, in that case an empty array is returned.